Scala 3 broken on oraclelinux

```
 > [ 3/10] RUN   case 3.0.0-RC2 in     "3"*) URL=https://github.com/lampepfl/dotty/releases/download/3.0.0-RC2/scala3-3.0.0-RC2.tar.gz SCALA_DIR=/usr/share/scala3-3.0.0-RC2 ;;     *) URL=https://downloads.typesafe.com/scala/3.0.0-RC2/scala-3.0.0-RC2.tgz SCALA_DIR=/usr/share/scala-3.0.0-RC2 ;;   esac &&   curl -fsL $URL | tar xfz - -C /usr/share &&   mv $SCALA_DIR /usr/share/scala &&   chown -R root:root /usr/share/scala &&   chmod -R 755 /usr/share/scala &&   ln -s /usr/share/scala/bin/* /usr/local/bin &&   case 3.0.0-RC2 in     "3"*) echo "@main def main = println(util.Properties.versionMsg)" > test.scala ;;     *) echo "println(util.Properties.versionMsg)" > test.scala ;;   esac &&   scala test.scala && rm test.scala:
#6 4.980 /usr/share/scala/bin/common: line 136: find: command not found
#6 4.982 /usr/share/scala/bin/common: line 136: find: command not found
#6 4.984 /usr/share/scala/bin/common: line 136: find: command not found
#6 4.987 /usr/share/scala/bin/common: line 136: find: command not found
#6 4.989 /usr/share/scala/bin/common: line 136: find: command not found
#6 4.992 /usr/share/scala/bin/common: line 136: find: command not found
#6 4.994 /usr/share/scala/bin/common: line 136: find: command not found
#6 4.997 /usr/share/scala/bin/common: line 136: find: command not found
#6 4.999 /usr/share/scala/bin/common: line 136: find: command not found
#6 5.002 /usr/share/scala/bin/common: line 136: find: command not found
#6 5.004 /usr/share/scala/bin/common: line 136: find: command not found
#6 5.007 /usr/share/scala/bin/common: line 136: find: command not found
#6 5.010 /usr/share/scala/bin/common: line 136: find: command not found
#6 5.032 /usr/share/scala/bin/common: line 136: find: command not found
#6 5.034 /usr/share/scala/bin/common: line 136: find: command not found
#6 5.037 /usr/share/scala/bin/common: line 136: find: command not found
#6 5.041 /usr/share/scala/bin/common: line 136: find: command not found
#6 5.044 /usr/share/scala/bin/common: line 136: find: command not found
#6 5.047 /usr/share/scala/bin/common: line 136: find: command not found
#6 5.049 /usr/share/scala/bin/common: line 136: find: command not found
#6 5.052 /usr/share/scala/bin/common: line 136: find: command not found
#6 5.056 /usr/share/scala/bin/common: line 136: find: command not found
#6 5.058 /usr/share/scala/bin/common: line 136: find: command not found
#6 5.061 /usr/share/scala/bin/common: line 136: find: command not found
#6 5.063 /usr/share/scala/bin/common: line 136: find: command not found
#6 5.065 /usr/share/scala/bin/common: line 136: find: command not found
#6 5.352 Error: Could not find or load main class dotty.tools.scripting.Main
#6 5.352 Caused by: java.lang.ClassNotFoundException: dotty.tools.scripting.Main
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -41,10 +41,6 @@ jobs:
             javaTag: '8u282'
             dockerContext: 'debian'
             baseImageTag: '8u282-jdk-buster'
-        exclude:
-          # scala 3.x broken on oraclelinux?
-          - scalaVersion: 3.0.0-RC2
-            javaTag: 11.0.10-oraclelinux8
     steps:
     - uses: actions/checkout@v1
     - name: Get latest SBT version

--- a/oracle/Dockerfile
+++ b/oracle/Dockerfile
@@ -18,6 +18,9 @@ ENV USER_ID ${USER_ID:-1001}
 ARG GROUP_ID 
 ENV GROUP_ID ${GROUP_ID:-1001}
 
+# Install findutils since find is required for scala 3
+RUN microdnf install findutils
+
 # Install sbt
 RUN \
   curl -fsL "https://github.com/sbt/sbt/releases/download/v$SBT_VERSION/sbt-$SBT_VERSION.tgz" | tar xfz - -C /usr/share && \

